src/get-started/data-proofs.md

```diff
@@ -41,32 +41,29 @@ Add necessary dependencies to `Cargo.toml` in the project directory:
 
 ```toml
 [dependencies]
-exonum = "0.9.0"
-exonum-configuration = "0.9.0"
-serde = "1.0"
-serde_derive = "1.0"
-failure = "=0.1.1"
+exonum = "0.10.2"
+exonum-derive = "0.10.0"
+exonum-configuration = "0.10.1"
+serde = "1.0.0"
+serde_derive = "1.0.0"
+failure = "0.1.5"
+protobuf = "2.2.0"
 ```
 
-Then import said dependencies into `src/lib.rs` file. We will use them to fetch
-the necessary structures, functions, types, traits etc.:
+Then we need to import macro from dependencies into `src/lib.rs` file.
 
 ```rust
 #[macro_use]
-extern crate exonum;
-#[macro_use]
-extern crate failure;
-extern crate serde;
-#[macro_use]
 extern crate serde_derive;
 ```
 
-For convenience reasons we decided to divide the code into 4 submodules, each
+For convenience reasons we decided to divide the code into 5 submodules, each
 corresponding to a certain part of the
-service business logic. Let’s announce them below:
+service business logic and one is for protobuf structures. Let’s announce them below:
 
 ```rust
 pub mod api;
+pub mod proto;
 pub mod schema;
 pub mod transactions;
 pub mod wallet;
```

Comment:
Shouldn't we specify all required imports? It relates to tutorial in general.

Reply:
Added imports for all files.